core/helper/model_utils.py

```diff
@@ -36,3 +39,34 @@ def create_eq_query(field, value):
 
 def any_queries(queries: Iterable[Q]):
     return functools.reduce(lambda a, b: a | b, queries, Q())
+
+
+def next_seq_safe(seq_name):
+    from django.db import connection
+
+    cursor = connection.cursor()
+    nextval_sql = f"select nextval('{seq_name}')"
+    try:
+        cursor.execute(nextval_sql)
+    except django.db.utils.ProgrammingError as e:
+        if f'"{seq_name}" does not exist' not in str(e):
+            raise e
+
+        init_val = settings.EMLO_SEQ_VAL_INIT.get(seq_name, None)
+        if init_val is None:
+            logging.debug(f'init val of [{seq_name}] not found in EMLO_SEQ_VAL_INIT')
+            init_val = 1
+
+        cursor.execute(f"CREATE SEQUENCE {seq_name} start with {init_val} ")
+        cursor.execute(nextval_sql)
+
+    result = cursor.fetchone()
+    return result[0]
+
+
+def default_current_timestamp():
```

Comment:
why not set default=django.utils.timezone.now when you define the field?

Reply:
yes, if django.utils.timezone.now is better, I will change it to
```
def default_current_timestamp():
   return django.utils.timezone.now()
```

Reply:
sure, but why have the utility function at all, why not just put the method directly?